e2eshark/run.py

```diff
@@ -458,10 +470,22 @@ def runCodeGeneration(
 
 
 # ∣input−other∣ ≤ atol + rtol × ∣other∣
-# returns (atol, rtol)
-# Pytorch torch.testing:
-def getTolerances(args):
-    return (args.atol, args.rtol)
+# Pytorch torch.testing defaults:
+#    bf16: atol=1e-05, rtol=1.6e-02
+#    fp16: atol=1e-05, rtol=1e-03
+#    fp32: atol=1e-05, rtol=1.3e-06
+def getTolerances(args, torchdtype):
+    atol, rtol = (1e-05, 1.3e-06)
+    if args.tolerance:
+        atol, rtol = tuple(args.tolerance)
+    elif torchdtype == torch.bfloat16:
+        atol, rtol = (1e-02, 1e-01)
+    elif torchdtype == torch.float16:
+        atol, rtol = (1e-04, 1e-03)
+    else:
+        atol, rtol = (1e-04, 1e-04)
+    return (atol, rtol)
```

Comment:
can simplify control flow here
```suggestion
    if args.tolerance:
        return tuple(args.tolerance)
    if torchdtype == torch.bfloat16:
        return (1e-02, 1e-01)
    if torchdtype == torch.float16:
        return (1e-04, 1e-03)
    return (1e-04, 1e-04)
```

Reply:
Sorry had missed that. Will do in next PR.